src/index.js

```diff
@@ -195,7 +195,12 @@ export default function (babel) {
       if (!tailPath) continue;
 
       if (tailPath.isExpressionStatement()) {
-        tailPath.replaceWith(getNewNode(tailPath.node.expression, tailPath));
+        const expression = tailPath.node.expression;
+        if (t.isAwaitExpression(expression)) {
```

Comment:
I think it may be a good idea to push this out to the `getNewNode` function passed in from `addImplicitReturns`. Other consumers of the `transformTails` API may actually want to see the entire `AwaitExpression`.